commands/proposal_submission.go

```diff
@@ -1697,6 +1697,23 @@ func isBindingMatchingSpecFilters(spec *vegapb.DataSourceDefinition, bindingProp
 	return bindingPropertyFound
 }
 
+func checkCompositePriceBinding(binding *vegapb.SpecBindingForCompositePrice, definition *vegapb.DataSourceDefinition, property string) Errors {
+	errs := NewErrors()
+	if binding != nil {
+		if len(binding.PriceSourceProperty) == 0 {
+			errs.AddForProperty(property, ErrIsRequired)
+		} else {
```

Comment:
nit-pick: `} else if !isBindingMatchingSpec() {` for 1 less level of indentation.